demos/benchmark/python/README.md

```diff
@@ -94,11 +95,9 @@ The client is able to download the metadata of the served models. If you are
 unsure which models and versions are served and what status they have, you can
 list this information by specifying the `--list_models` switch (also a short
 form `-l` is available):
-```
+```bash
 docker run --network host benchmark_client -a localhost -r 30002 --list_models
 
-OVMS benchmark client 1.17
-NO_PROXY=localhost no_proxy=localhost python3 /ovms_benchmark_client/main.py -a localhost -r 30002 --list_models
```

Comment:
It was intentional since I couldn't compare NO_PROXY line without getting an error (there are other values than localhost when I run it on my machine). But after checking again on builder04 there is no error so I'm reverting that change.